### PR TITLE
Feature / Font size selector on PDF export

### PR DESCRIPTION
**Pull request by chelsouse**
PR for issue #1575

### CODE REVIEW

File: src/components/Settings/Export/Export.component.js [hunk 1]
```diff
@@ -39,6 +39,7 @@ class Export extends React.Component {
     this.state = {
       exportSingleBoard: '',
       exportAllBoard: '',
+      exportAllBoardSize: '16',
```
Comment (RodriSanchez1) [inline src/components/Settings/Export/Export.component.js]: A better variable name can be `labelFontSize` ,`tileLabelFontSize` or something more descriptive

File: src/components/Settings/Export/Export.component.js [hunk 2]
```diff
@@ -282,6 +296,44 @@ class Export extends React.Component {
...
+                        {intl.formatMessage(messages.size)}
+                      </InputLabel>
+                      <Select
+                        labelId="export-all-select-label-size"
+                        id="export-all-select-size"
+                        autoWidth={false}
+                        value={this.state.exportAllBoardSize}
+                        onChange={this.handleSizeChange}
+                      >
+                        <MenuItem value="12">Small</MenuItem>
```
Comment (RodriSanchez1) [inline src/components/Settings/Export/Export.component.js]: Add ´Small´, ´Medium´ and ´Large´ words to the ´Export.messages.js´ to get the translations 

Use this values instead:
 - Small: 9
 - Medium: 12
 - Large: 16

Remember to update the default value on the state definition! Line 42

File: src/components/Settings/Export/Export.component.js [hunk 3]
```diff
@@ -282,6 +296,44 @@ class Export extends React.Component {
                   </div>
                 </ListItemSecondaryAction>
               </ListItem>
+              <Divider />
```
Comment (RodriSanchez1) [inline src/components/Settings/Export/Export.component.js]: Im thinking we can improve it. Take a look at this: 
image

What do you think? You can investigate on the `Settings.component.js` . The settings section use  the UI proposted

File: src/components/Settings/Export/Export.helpers.js [hunk 4]
```diff
@@ -858,6 +896,9 @@ export async function pdfExportAdapter(boards = [], intl, picsee = false) {
     }
   };
   if (picsee) {
+    //exportAllBoardSize = exportAllBoardSize - 4;
```
Comment (RodriSanchez1) [inline src/components/Settings/Export/Export.helpers.js]: remove the comment!

File: src/components/Settings/Export/Export.helpers.js [hunk 5]
```diff
@@ -598,6 +631,7 @@ const addTileToGrid = async (
   const labelData = {
     text: label,
     alignment: 'center',
+    fontSize: exportAllBoardSize,
```
Comment (RodriSanchez1) [inline src/components/Settings/Export/Export.helpers.js]: See line 643  - 
`if (imageData.width <= 37) {
    labelData.fontSize = 7;
  } else if (imageData.width <= 40) {
    labelData.fontSize = 8;
  } else if (imageData.width <= 45) {
    labelData.fontSize = 9;
  }`

We need to manage also this!

**Review by RodriSanchez1 — CHANGES_REQUESTED** [review]
@chelsouse Nice work!! Take a look at the comments that i made! Let me now if you have some doubts!

**Comment by chelsouse** [comment]
Hello @RodriSanchez1

I've implemented the requested changes. Please let me know, if it needs any futher changes.

**Review by RodriSanchez1 — APPROVED** [review]
@chelsouse Nice workk!  I made a few changes, check it out! 

I want to give you a little advice, in your future commits try to implement atomic commits. This way, your work will be more detailed. An also always add a PR description!

See:
_[What's an atomic git commit?](https://dev.to/samuelfaure/how-atomic-git-commits-dramatically-increased-my-productivity-and-will-increase-yours-too-4a84)
Working with atomic git commits means your commits are of the smallest possible size. Each commit does one, and only one simple thing, that can be summed up in a simple sentence.
The amount of code change doesn't matter. It can be a letter or it can be a hundred thousand lines, **but you should be able to describe the change with one simple short sentence.**_

Hope you want continue contributing on Cboard! 
Regards, Rodri!

**Comment by chelsouse** [comment]
Hello @RodriSanchez1 

Thanks for your help and advice on atomic commits! It's my first PR, and I'll definitely keep your suggestions in mind. Appreciate the guidance!

**Pull request merged**